book/scheduling-and-threading.md

```diff
@@ -93,137 +44,130 @@ class Task:
         self.args = args
         self.__name__ = "task"
 
-    def __call__(self):
+    def run(self):
         self.task_code(*self.args)
         self.task_code = None
         self.args = None
 ```
 
-Also define a new `TaskRunner` class to manage the task queues and run them. It
-will have a list of `Task`s, a method to add a `Task`, and a method to run once
-through the event loop. Implement a simple scheduling heuristic in
-`run` that executes one task each time through, if there is one to run.
+The point of a task is that it can be created at one point, and then
+run at some later point by a task runner of some kind.[^event-loop] In
+our browser, the task runner will store tasks in a first-in first-out
+queue:
 
-``` {.python expected=False}
+[^event-loop]: Event loops, which we discussed in [Chapter
+2](graphics.md#eventloop) and [Chapter
+11](visual-effects.md#sdl-creates-the-window) are one kind of task
+runner, where the tasks to run are provided by the operating system.
```

Comment:
What other kind of task runners than event loops did you have in mind?

Reply:
Fixed, I meant to refer to the two specific event loops we've seen. Changed to:

> The event loops we discussed in Chapter 2 and Chapter 11 are task runners, where the tasks to run are provided by the operating system.